{
  "uid" : "6b70a33cea7eb539",
  "name" : "test_get_allowance[Polygon (MATIC)]",
  "fullName" : "tests.api_tests.tests_de_bridge#test_get_allowance",
  "historyId" : "77fa68db098c403c41c4c8f2dd7e20a0",
  "time" : {
    "start" : 1685610404194,
    "stop" : 1685610404584,
    "duration" : 390
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: Ожидался status = 200, пришел status = 400. Тело ответа: \n{\"ok\":false,\"data\":null,\"error\":\"No available RPC's for ETH\"}.",
  "statusTrace" : "network = 'Polygon (MATIC)'\n\n    @pytest.mark.parametrize(\"network\", networks)  # todo разобрать ошибку\n    def test_get_allowance(network):  # \"error\":\"No available RPC's for ETH\"\n        net = de_bridge_keys_data()[network].get('net')\n        not_native_token = de_bridge_keys_data()[network].get('notNativeToken')\n        owner = de_bridge_keys_data()[network].get('owner')\n>       DeBridge().get_allowance(net=net, not_native_token=not_native_token, owner=owner)\n\ntests/api_tests/tests_de_bridge.py:49: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <serviceCalls.debrdige_service.DeBridge object at 0x10ea6b700>\nnet = 'polygon', not_native_token = '0xa1428174f516f527fafdd146b883bb4428682737'\nowner = '0xd5ed26d93129a8b51ac54b40477327f6511824b6'\n\n    def get_allowance(\n            self, net, not_native_token,\n            owner):  # todo: написать метод для урла с параметрами (параметры затребовать с Егора). в конфлюенс не\n        # указан параметр net валится с 504 ошибкой. сделать после исправления\n        response = requests.get(f\"{self.url}getAllowance?net={net}&tokenAddress={not_native_token}&owner={owner}\")\n        logger.info(f\"Отправлен запрос на роут - {response.url}\")\n        logger.info(response.text)\n>       assert response.status_code == 200, f\"Ожидался status = {200}, \" \\\n                                            f\"пришел status = {response.status_code}. \" \\\n                                            f\"Тело ответа: \\n{response.text}.\"\nE       AssertionError: Ожидался status = 200, пришел status = 400. Тело ответа: \nE       {\"ok\":false,\"data\":null,\"error\":\"No available RPC's for ETH\"}.\n\nserviceCalls/debrdige_service.py:79: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: Ожидался status = 200, пришел status = 400. Тело ответа: \n{\"ok\":false,\"data\":null,\"error\":\"No available RPC's for ETH\"}.",
    "statusTrace" : "network = 'Polygon (MATIC)'\n\n    @pytest.mark.parametrize(\"network\", networks)  # todo разобрать ошибку\n    def test_get_allowance(network):  # \"error\":\"No available RPC's for ETH\"\n        net = de_bridge_keys_data()[network].get('net')\n        not_native_token = de_bridge_keys_data()[network].get('notNativeToken')\n        owner = de_bridge_keys_data()[network].get('owner')\n>       DeBridge().get_allowance(net=net, not_native_token=not_native_token, owner=owner)\n\ntests/api_tests/tests_de_bridge.py:49: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <serviceCalls.debrdige_service.DeBridge object at 0x10ea6b700>\nnet = 'polygon', not_native_token = '0xa1428174f516f527fafdd146b883bb4428682737'\nowner = '0xd5ed26d93129a8b51ac54b40477327f6511824b6'\n\n    def get_allowance(\n            self, net, not_native_token,\n            owner):  # todo: написать метод для урла с параметрами (параметры затребовать с Егора). в конфлюенс не\n        # указан параметр net валится с 504 ошибкой. сделать после исправления\n        response = requests.get(f\"{self.url}getAllowance?net={net}&tokenAddress={not_native_token}&owner={owner}\")\n        logger.info(f\"Отправлен запрос на роут - {response.url}\")\n        logger.info(response.text)\n>       assert response.status_code == 200, f\"Ожидался status = {200}, \" \\\n                                            f\"пришел status = {response.status_code}. \" \\\n                                            f\"Тело ответа: \\n{response.text}.\"\nE       AssertionError: Ожидался status = 200, пришел status = 400. Тело ответа: \nE       {\"ok\":false,\"data\":null,\"error\":\"No available RPC's for ETH\"}.\n\nserviceCalls/debrdige_service.py:79: AssertionError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "51669de053dec027",
      "name" : "log",
      "source" : "51669de053dec027.txt",
      "type" : "text/plain",
      "size" : 332
    } ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "tests.api_tests"
  }, {
    "name" : "suite",
    "value" : "tests_de_bridge"
  }, {
    "name" : "host",
    "value" : "MacBook-Air-Anna.local"
  }, {
    "name" : "thread",
    "value" : "16727-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.api_tests.tests_de_bridge"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "network",
    "value" : "'Polygon (MATIC)'"
  } ],
  "links" : [ ],
  "hidden" : true,
  "retry" : true,
  "extra" : {
    "categories" : [ ],
    "tags" : [ ]
  },
  "source" : "6b70a33cea7eb539.json",
  "parameterValues" : [ "'Polygon (MATIC)'" ]
}